Integrations/ArcSightESMv2/ArcSightESMv2.py

```diff
@@ -82,9 +102,14 @@ def beautifully_json(d, depth=0):
                 if isinstance(value, dict):
                     beautifully_json(value, depth + 1)
                 elif value in NONE_VALUES:
-                    d.pop(key, None)
-                elif key in IP_FIELDS and isinstance(value, int):
-                    d[key] = int_to_ip(value)
+                    if remove_nones:
+                        d.pop(key, None)
+                    else:
+                        d[key] = 'None'
+
+                elif key in IP_FIELDS:
+                    key = 'decodedAddress' if key == 'addressAsBytes' else key
```

Comment:
why do we change the key here?

Reply:
we are adding the decoded output into a new key to keep backward compatibility